src/form-message.scss

```diff
@@ -9,32 +9,22 @@ $block: #{$fd-namespace}-form-message;
 
 .#{$block} {
   @include fd-reset();
+  @include fd-ellipsis();
 
-  z-index: 3;
-  clear: both;
-  height: auto;
-  width: fit-content;
+  padding: 0.5rem;
   min-width: 6rem;
   max-width: 22rem;
-  padding: 0.5rem;
-  margin-top: -0.25rem;
+  width: 100%;
   font-size: var(--sapFontSmallSize);
   font-weight: normal;
   line-height: normal;
   color: var(--sapTextColor);
-  overflow: hidden;
-  text-overflow: ellipsis;
-  border-radius: 0 !important;
   white-space: pre-line !important;
 
   @include fd-rtl() {
     right: 0;
   }
 
-  &--compact {
-    margin-top: -0.1875rem;
```

Comment:
Where are you taking care of the compact version?

Reply:
<img width="1008" alt="Screen Shot 2020-02-12 at 8 48 40 PM" src="https://user-images.githubusercontent.com/39598672/74393836-30314a00-4dd9-11ea-89b6-59e2dab84cc7.png">

no need to have a specific value for compact. -0.25rem is handling both cases. I included an example with compact input in the documentation